ci/update-requirements.py

```diff
@@ -42,16 +48,19 @@ def main():
 
     openvino_dir = Path(os.environ['INTEL_OPENVINO_DIR'])
 
-    pip_compile('ci/requirements-ac.txt',
+    def pc(target, *sources):
```

Comment:
maybe partial function usage will be better?

Reply:
Yeah, good point. Updated.